README.md

```diff
@@ -69,6 +69,21 @@ You can install and run the NativeScript CLI on Windows or OS X.
 
 > On Windows systems, you can develop, build, and deploy NativeScript projects that target Android.
 
+### Setup Script
+To quickly set up your system for the latest NativeScript CLI, paste the following PowerShell script in the **command prompt** and hit `Enter`:
+```cmd
+@powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+```
+
+Alternatively, your can paste the following PowerShell setup script in a **Windows PowerShell console** and hit `Enter`:
+```PowerShell
+start-process -FilePath PowerShell.exe -Verb Runas -Wait -ArgumentList "-NoProfile -ExecutionPolicy Bypass -Command iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+```
+> Both scripts require that you have **.NET 4.0 or later** installed on your system.
```

Comment:
Fair enough - will fix it, shall I also fix the link inside the PowerShell setup script?